CHANGELOG.md

```diff
@@ -4,6 +4,9 @@
 * Removed package list of 2.10 null-safety experiments.
 * Update dependencies to latest.
 * Migration to null-safety.
+* **BREAKING CHANGES**
+  * Deprecate the `SCREAMING_CAPS` versions of the `LicenseNames` constants. 
+    Use the newly introduced `lowerCamelCase` versions instead.
```

Comment:
How is this breaking? You just deprecated the old names, you didn't remove them.

Reply:
@isoos suggested adding the breaking change descriptor in a previous review. Perhaps due to plans to remove them as part of the release in a follow up or just to make it more clear for those reading the changelog that they need to migrate. 

Let me know if you want me to instead delete the breaking change mention in a new PR.

Reply:
Yes, this was after we've decided to do a breaking release with pana's null-safety migration. I'll create a PR that removes the old values, and also hides some internals that were exposed as public API.

Reply:
Sounds great @isoos, thank you!